### PR TITLE
Add Zotero link integration

### PR DESCRIPTION
**Pull request by MunGell**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: https://github.com/MunGell/obsidian-zotero

## Release Checklist
- [x] I have tested the plugin on
  - [ ]  Windows
  - [x]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [ ] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by liamcain** [comment]
https://github.com/MunGell/obsidian-zotero/blob/7f17036c513bbdc5ef9631d49e2cf0197a5a9c48/src/ZoteroConnector.ts#L13 you can turn this into a [`getter`](https://www.typescriptlang.org/docs/handbook/2/classes.html#getters--setters) instead of setting it in the constructor. That way the value can be derived from `this.settings` without needing to get recomputed.

https://github.com/MunGell/obsidian-zotero/blob/7f17036c513bbdc5ef9631d49e2cf0197a5a9c48/src/ZoteroPlugin.ts#L32 if you fix the above, then you don't need to call `init` from within saveSettings

https://github.com/MunGell/obsidian-zotero/blob/7f17036c513bbdc5ef9631d49e2cf0197a5a9c48/src/ZoteroItem.ts#L54 Before doing this, you should call [`sanitizeHTMLToDom`](https://github.com/obsidianmd/obsidian-api/blob/9bb30633586c6649474b2affdf5b87efdd86498f/obsidian.d.ts#L2726). That way you aren't leaving this method exposed to a security risk.

**Comment by MunGell** [comment]
Thanks for review @liamcain 

Your suggestions were addressed in the code

File: community-plugins.json [hunk 1]
```diff
@@ -4272,5 +4272,12 @@
         "author": "Zsolt Viczian",
         "description": "ExcaliBrain is inspired by TheBrain and Breadcrumbs. It is an interactive, structured mind-map of your Obsidian Vault generated based on the folders and files in your Vault by interpreting the links, dataview fields, tags and YAML front matter in your markdown files.",
         "repo": "zsviczian/excalibrain"
+    },
+    {
+        "id": "obsidian-zotero",
+        "name": "Zotero",
```
Comment (liamcain) [inline community-plugins.json:4278]: Sorry I didn't notice this earlier, but–

I think you might want to change the plugin name (something like Zotero Link is fine). Having it as just "Zotero" could imply it's an official product of the Zotero team and might cause a bit of confusion from users.

Comment (MunGell) [reply community-plugins.json]: Thank you for highlighting this, I've renamed it to ZoteroBridge here and in the repository now.

**Comment by MunGell** [comment]
Unfortunately, I have to close this PR for the moment. I am working on a refactored version of the plugin that might have a different name (Zotero Link), so it is better to wait until it will be done to avoid user confusion. Will open new PR soon.

**Pull request closed without merging**